step-certificates/values.yaml

```diff
@@ -339,6 +339,18 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
+# Configure extra volumes and volume mounts as YAML expressions.
+# And example that will mount an extra volume would be:
+# volumes:
+# - name: custom-ca
+#   configMap:
+#     name: custom-ca-certs
+# volumeMounts:
+# - name: custom-ca
+#   mountPath: /etc/ssl/certs/custom-ca.pem
+volumes: []
+volumeMounts: []
```

Comment:
I noticed above that we have a [`podExtraLabels`](https://github.com/smallstep/helm-charts/blob/master/step-certificates/values.yaml#L59), and I wonder for this if it should be `extraVolumes` and `extraVolumeMounts`.

Reply:
We were already using this in `step-issuer` so I kept the same name, but I can change it. Let me look for a common name for it in other deployments.